etl/steps/data/garden/cedlas/2024-03-08/sedlac.meta.yml

```diff
@@ -0,0 +1,749 @@
+        description_short: Percentage of population living in households with an income or consumption per person below <<table.lower()>>
+        description_key:
+          - "{definitions.poverty_lines}"
+          - "{definitions.ppps}"
+          - "{definitions.welfare_equivalence}"
+        unit: '%'
+        short_unit: '%'
+        display:
+          name: <<table>> - Share of population in poverty
+          <<: *common-display
+        presentation:
+          title_public: <<table>> - Share of population in poverty
+
+      national_poverty_gap_index:
+        title: <<table>> - Poverty gap index (Spell <<survey_number>>)
+        description_short: The poverty gap index is a poverty measure that reflects both the prevalence and the depth of poverty. It is calculated as the share of population in poverty multiplied by the average shortfall from the poverty line (expressed as a % of the poverty line).
+        description_key:
+          - "{definitions.poverty_lines}"
+          - "{definitions.ppps}"
+          - "{definitions.welfare_equivalence}"
+        unit: '%'
+        short_unit: '%'
+        display:
+          name: <<table>> - Poverty gap index
+          <<: *common-display
+        presentation:
+          title_public: <<table>> - Poverty gap index
+
+      national_fgt2:
+        title: <<table>> - Poverty severity index (Spell <<survey_number>>)
+        description_short: The squared poverty gap index (FGT(2)) is a poverty measure that reflects both the prevalence, the depth and the distribution of poverty. A poverty severity of 5 percent means that the poorest people are 5 percent worse off compared to poor people on average.
+        description_key:
+          - "{definitions.poverty_lines}"
+          - "{definitions.ppps}"
+          - "{definitions.welfare_equivalence}"
+        unit: '%'
+        short_unit: '%'
+        display:
+          name: <<table>> - Poverty severity index
+          <<: *common-display
+        presentation:
+          title_public: <<table>> - Poverty severity index
+
+      gini:
+        title: Gini coefficient (<<table>>) (Spell <<survey_number>>)
+        description_short: The [Gini coefficient](#dod:gini) measures inequality on a scale from 0 to 1. Higher values indicate higher inequality.
+        description_key:
+          - "{definitions.welfare_equivalence}"
+        unit: ''
+        short_unit: ''
+        display:
+          name: Gini coefficient (<<table>>)
+          <<: *common-display
+        presentation:
+          title_public: Gini coefficient (<<table>>)
+
+      theil:
+        title: Theil index (<<table>>) (Spell <<survey_number>>)
+        description_short: The Theil index is part of a larger family of measures referred to as the General Entropy class of inequality measures. It measures an entropic distance the population is away from the ideal egalitarian state of everyone having the same income.
```

Comment:
wonder if there is a way to make the description_short less technical - e.g., "The Theil index looks at how different people's incomes are from an ideal situation where everyone earns the same amount. Higher values mean..."